src/collective/easyform/interfaces/actions.py

```diff
@@ -118,7 +105,9 @@ class IActionEditForm(z3c.form.interfaces.IEditForm):
 
 
 class IAction(Schema, zope.schema.interfaces.IField):
-    directives.omitted("required", "order", "default", "missing_value", "readonly")
+    directives.omitted(
+        "description", "required", "order", "default", "missing_value", "readonly"
```

Comment:
IMO we really do not need the "description" field.
I also want to hide the ``__name__`` (shortname for programmatic access) for normal users (should be editable by managers). The shortname should be generated automatically unless needed.

Reply:
It's auto-updated by javascript after filling the "title". Thanks for the reminder, have to check this.